docker/Dockerfile.cpu

```diff
@@ -3,23 +3,25 @@ FROM ubuntu:18.04
 ENV PYTHONUNBUFFERED TRUE
 
 RUN apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y \
-    build-essential \
-    libatlas-base-dev \
-    libopencv-dev \
-    graphviz \
-    python-dev \
+    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
+    fakeroot \
+    ca-certificates \
+    dpkg-dev \
+    g++ \
```

Comment:
Why do we need this?

Reply:
This is much smaller than build-enssential

Reply:
Did we need build-essential for opencv? Isn't g++ a compiler? Would we need it?